source/ShopPanel.cpp

```diff
@@ -652,10 +771,12 @@ void ShopPanel::DoFind(const string &text)
 
 int64_t ShopPanel::LicenseCost(const Outfit *outfit, bool onlyOwned) const
 {
+	// onlyOwned represents that `outfit` is being transferred from Cargo or Storage
```

Comment:
Maybe just rename it to some more descriptive name?

Reply:
I mean, it's probably OK.  The comment helps, I think, because it clarifies that in this context the source of the item will be _only_ from the player _owned_ stores.

This reminds me, I didn't add automatic license buying into the buy to Cargo... so that's a bug.

Reply:
- [ ] > This reminds me, I didn't add automatic license buying into the buy to Cargo... so that's a bug.

Reply:
@TomGoodIdea -- do you know how the licensing is supposed to work with outfits?  Is there an example/test setup that I can use to test the purchase of an outfit that requires a license?  Are licenses inherently available (seem to be) anywhere that an outfit requiring a license is available?

Reply:
> @TomGoodIdea -- do you know how the licensing is supposed to work with outfits? Is there an example/test setup that I can use to test the purchase of an outfit that requires a license? Are licenses inherently available (seem to be) anywhere that an outfit requiring a license is available?

nevermind -- I happened upon this in my actual game for the first time just now :-)